src/countAnimals.js

```diff
@@ -1,7 +1,8 @@
 const data = require('../data/zoo_data');
 
-function countAnimals(species) {
+function countAnimals() {
```

Comment:
```suggestion
function countAnimals(species) {
```
Acho que da pra manter o parâmetro

Reply:
Boa, vou fazer isso, mas acho que como ele vai receber um objeto que pode receber specie e genre como chave, talvez um nome diferente seja melhor.